optimum/exporters/tasks.py

```diff
@@ -637,9 +637,11 @@ def determine_framework(model: str, framework: str = None) -> str:
             4. If could not infer, use available framework in environment, with priority given to PyTorch.
 
         Args:
-            model (`str`):
+            model_name (`str`):
```

Comment:
Why `model_name` instead of `pretrained_model_name_or_path` like in transformers? Can we use a path or only models hosted on the hub?